### ISSUE TITLE
Describe units in documentation

### ISSUE THREAD

**Issue by mwhitworth** [Stratus3D/eflambe#42]
Despite my diving through the `eflambe` codebase and `erlang:trace` functionality, it's not really possible to say with confidence what unit the data generated is actually in. (Is it reductions?) It would be clearer to state as part of the initial description what the fundamental unit of measurement is in

**Comment by Stratus3D** [comment]
Yes, this is really important! They aren't in reductions. Currently the unit output by eflambe should be microseconds, but I want to add support for reductions as well.

**Comment by Stratus3D** [comment]
Related - https://github.com/Stratus3D/eflambe/issues/11
